src/Microsoft/SqlServer/Management/SqlEnum/xml/CloudDependency.sql

```diff
@@ -480,6 +497,25 @@ begin
 			join sys.tables as tbl on tbl.object_id = fk.referenced_object_id
 			where @iter_no = t.rank and t.object_type = @u and (t.object_svr IS null and t.object_db = db_name())
 		set @rows = @rows + @@rowcount
+                
+                -- Partition Schemes referenced by tables/views
+                insert #t1 (object_id, object_name, object_db, object_type, relative_id, relative_name, relative_schema, relative_db, relative_type, schema_bound, rank)
+                    select ps.data_space_id, ps.name, t.object_db, @part_sch, t.object_id, t.object_name, t.object_schema, t.object_db, t.object_type, 1, @iter_no + 1
+                    from #t1 as t
```

Comment:
looks like this file uses tabs for SQL indent.
Switch to tabs for now then I can reformat the entire doc to spaces in another change.